README.md

```diff
@@ -23,30 +23,25 @@
 
 ----
 
-## Important Announcement: Spyder needs your support!
-
-Since mid-November 2017, [Anaconda, Inc](https://www.anaconda.com/) has
-stopped funding Spyder development, after doing so for the past 18
-months. Therefore, without additional funds, development will shift to
-maintaining Spyder 3 at a slower pace than before, while working
-toward an eventual Spyder 4 feature release sometime in the future.
-
-However, with your contribution of effort and funding, we will be able to
-both continue to maintain Spyder 3 at a faster pace, and fund development of
-new features for Spyder 4 (such as a major overhaul to improve
-code completion, and a much-improved new debugger, both of which you've been
-requesting) at a greatly accelerated rate.
-
-There are many ways to [help with development](
+## Help support the community-developed scientific IDE!
```

Comment:
Let's change this to

> Help support Spyder, the community-...

to make things easier to understand

Reply:
Sure; done.